projects/laji/test/+vihko/submissions.po.ts

```diff
@@ -0,0 +1,39 @@
+import { Locator, Page } from '@playwright/test';
+
+export class SubmissionsPage {
+  private $datatable = (this.$container ? this.$container : this.page).locator('laji-own-datatable');
+
+  constructor(
+    private page: Page,
+    private $container?: Locator
+  ) {}
+
+  navigateTo() {
+    return this.page.goto('/vihko/ownSubmissions');
+  }
+
+  datatable = {
+    $container: this.$datatable,
+    $rows: this.$datatable.locator('datatable-row-wrapper'),
+    waitUntilLoaded: () => this.$datatable.locator('.spinner').waitFor({state: 'hidden'}),
+    getRow: (name: string) => {
+      const $container = this.$datatable.locator('datatable-row-wrapper').filter({
+        has: this.page.locator(`span[title="${name}"]`)
+      });
+      return {
+        $container,
+        $buttons: $container.locator('.btn'),
+        $viewButton: $container.locator('.view-button'),
+        $editButton: $container.locator('.edit-button'),
+        $templateButton: $container.locator('.template-button'),
+        $downloadButton: $container.locator('.download-button'),
+        $deleteButton: $container.locator('.delete-button'),
+      };
+    },
```

Comment:
This doesn't work now for `submissions.spec.ts` and `tools.spec.ts`  using this and, because `submissionsPage.datatable.getRow(0)` tries to find a single row by a span having title `0`. There are multiple cells in different rows with `0` as value, so this doesn't work... I guess `0` was for index before?

```
expect.toBeVisible: Error: strict mode violation: locator('laji-own-datatable').locator('datatable-row-wrapper').filter({ has: locator('span[title="0"]') }).locator('.view-button') resolved to 14 elements:
    1) <button _ngcontent-laji-app-c3276983730="" class="btn bt…>…</button> aka getByRole('row', { name: ' Näytä  Muokkaa 07.02.2024 12:17 07.02.2024 Oulu 1 0 E2E Testing Retkilomake' }).getByRole('button').first()
    2) <button _ngcontent-laji-app-c3276983730="" class="btn bt…>…</button> aka getByRole('row', { name: ' Näytä  Muokkaa 07.02.2024 12:12 07.02.2024 Oulu 1 0 E2E Testing Retkilomake' }).getByRole('button').first()
    3) <button _ngcontent-laji-app-c3276983730="" class="btn bt…>…</button> aka getByRole('row', { name: ' Näytä  Muokkaa 06.02.2024 13:44 06.02.2024 Oulu 1 0 E2E Testing Retkilomake' }).getByRole('button').first()
    4) <button _ngcontent-laji-app-c3276983730="" class="btn bt…>…</button> aka getByRole('row', { name: ' Näytä  Muokkaa 06.02.2024 13:40 06.02.2024 Oulu 1 0 E2E Testing Retkilomake' }).getByRole('button').first()
    5) <button _ngcontent-laji-app-c3276983730="" class="btn bt…>…</button> aka getByRole('row', { name: ' Näytä  Muokkaa 06.02.2024 13:38 06.02.2024 Oulu 1 0 E2E Testing Retkilomake' }).getByRole('button').first()
    6) <button _ngcontent-laji-app-c3276983730="" class="btn bt…>…</button> aka getByRole('row', { name: ' Näytä  Muokkaa 06.02.2024 13:28 06.02.2024 Oulu 1 0 E2E Testing Retkilomake' }).getByRole('button').first()
    7) <button _ngcontent-laji-app-c3276983730="" class="btn bt…>…</button> aka getByRole('row', { name: ' Näytä  Muokkaa 06.02.2024 13:22 06.02.2024 Oulu 1 0 E2E Testing Retkilomake' }).getByRole('button').first()
    8) <button _ngcontent-laji-app-c3276983730="" class="btn bt…>…</button> aka getByRole('row', { name: ' Näytä  Muokkaa 06.02.2024 12:37 06.02.2024 Oulu 1 0 E2E Testing Retkilomake' }).getByRole('button').first()
    9) <button _ngcontent-laji-app-c3276983730="" class="btn bt…>…</button> aka getByRole('row', { name: ' Näytä  Muokkaa 06.02.2024 12:34 06.02.2024 Oulu 1 0 E2E Testing Retkilomake' }).getByRole('button').first()
    10) <button _ngcontent-laji-app-c3276983730="" class="btn bt…>…</button> aka getByRole('row', { name: ' Näytä  Muokkaa 06.02.2024 12:24 06.02.2024 Oulu 1 0 E2E Testing Retkilomake' }).getByRole('button').first()
    ...
```

Maybe `getRow()` should take a locator as input that targets a row, instead of it digging by a cell title content? Or two different functions, `getRowByIdx` and `getRowByCellContent`?

Reply:
Fixed, I didn't notice that the function was used in many places